examples/lambda/README.md

```diff
@@ -42,4 +42,24 @@ npm run cdk deploy
 
 The lambda does not set up a way to access itself externally, so to run it, you will have to go to `MomentoDockerLambda` in AWS Lambda and run a test.
 
-The lambda is set up to make set and get calls for the key 'key' in the cache 'cache'. You can play around with the code by changing the `docker/lambda/index.py` file. Remember to update `docker/lambda/aws_requirements.txt` file if you add additional Python dependencies.
\ No newline at end of file
+The lambda is set up to make set and get calls for the key 'key' in the cache 'cache'. You can play around with the code by changing the `docker/lambda/index.py` file. Remember to update `docker/lambda/aws_requirements.txt` file if you add additional Python dependencies.
+
+## Deploying the Momento Python Lambda as a Zip File on AWS Lambda
+
+Alternatively, we can deploy the Momento Python Lambda as a Zip File on AWS Lambda. We can do this using the `zip` directory in this example.
+
+Follow these steps to create the zip and deploy it to AWS Lambda using the AWS Management Console:
+
+1. Run `make dist` in the `zip` directory to package the lambda for upload as `dist.zip`.
+   :tipp: Check out the Makefile for important build steps to package for AWS Lambda.
+
+2. Create a new Lambda function by selecting "Author from scratch".
+3. Set the function name to `momento-lambda-demo`.
+4. Choose the runtime as `Python 3.8` (you can adjust this as desired).
+5. Select the architecture as `x86_64`.
+6. Click on "Create function" to create the Lambda function.
+7. In the "Code" tab, choose "Upload from the zip" as the code source.
```

Comment:
@eaddingtonwhite - do you have any input as to whether showing the steps to deploy via console will be sufficient for most users? In the one we just did for nodejs compression we showed how to deploy via `aws` cli and `sam`, but I don't have a good sense for whether users really need help with those parts once they have the zip part sorted.

Reply:
Yeah actually think console maybe better less moving parts or other steps people could mess up.

Reply:
okay gtk!

in that case, 🚢